pages/components/attention.html

```diff
@@ -170,25 +170,25 @@ <h2 class="mt-24 mb-16">Accessibility</h2>
         </p>
         <p>
           Attention element handles accessibility automatically by wrapping its slotted content with
-          a <code>div</code> with <code>role="tooltip"</code>, and setting an
-          <code>aria-describedby</code> attribute on the target element.
+          a <code>div</code> with either <code>role="tooltip"</code> for tooltip or <code>role="img"</code> for callout and popover, a default <code>aria-label</code>and setting an
+          <code>aria-details</code> attribute on the target element. The default <code>aria-label</code> also supports i18n.
         </p>
         <p>
           It is possible to tell assistive technologies to recognize only a part of Attention's text
-          content. To do that set <code>role="tooltip"</code> on the relevant text element nested in
+          content. To do that set the <code>role</code> and <code>aria-label</code> attributes on the relevant text element nested in
```

Comment:
You don't need to set `aria-label` on the referenced text element. As long as it has a `role`, screen reader will read its contents (text).

Reply:
Ah goood to know! 😄 I'll remove it then :)